### DISCUSSION TITLE
Unavailable Table of Content

### DISCUSSION THREAD

**Discussion by mingyuecall** [transitive-bullshit/nextjs-notion-starter-kit#245] [Q&A] [answered]
Hi there! I was wondering why there is no table of content when I have created more than 3 heading 1.

**Comment by transitive-bullshit** [comment, marked as answer]
1. make sure you're on a page where `showTableOfContents` is set to `true`
2. make sure the page has at least `minTableofContentsItems` headers
3. make sure you're viewing on a browser wide enough to display the toc
4. make sure you're using the latest version of the starter kit and `react-notion-x` dependencies

If you've done all of these, then post your notion page ID and I'll take a look.

**Reply by mingyuecall** [reply]
Dear Developer Thank you so much for replying to me. I think I can make sure that points 2, 3 & 4 are set right as I was forking the [Code here](https://github.com/transitive-bullshit/nextjs-notion-starter-kit) yesterday and am using Brave browser on Macbook. Unfortunately I am a code-knowledge-less person, and I wonder that do you mean `showTableOfContents` on my repository? I am sorry to take your time. I will then share my link and delete it after you get it. Thank you very much.

**Reply by transitive-bullshit** [reply]
Can you give me a notion ID I can test with?

By default, the TOC will only show up on pages that are children of collections, but you can change that if you go into the code.

**Reply by mingyuecall** [reply]
Dear Travis, due to the privacy protection I had emailed you two days ago, along with my website and notion vault. Could you please have a look? If it is not a tech issue caused by the repository here, I will then turn to my friends for help. Please forgive me for being ignorant of coding... 😊

**Reply by transitive-bullshit** [reply]
@MingChen2020 I'm not sure which email you're referring to.. can you send the public notion ID to travis@transitivebullsh.it?

**Reply by mingyuecall** [reply]
@transitive-bullshit transitive Yes Travis Thank you. This is the email I've referred to.

**Reply by transitive-bullshit** [reply]
@MingChen2020 I'm saying that I never received an email from you. Could you please re-send?

**Reply by transitive-bullshit** [reply]
I think it's disabled because you have the "full page width" setting turned on in notion.

If the page is too wide, then the TOC won't be shown on the side.

**Reply by mingyuecall** [reply]
Okay Travis Thank you so much. I will then have a try.

**Reply by mingyuecall** [reply]
Dear Travis. I have to say now it works very well after turning off the full width mode. Thank you so much. I hope this discussion help other people with the same issue.

**Reply by transitive-bullshit** [reply]
Awesome; glad it helped :)

**Reply by coderlfm** [reply]
Hello, I found that if the content of the article is `paste and sync` content, the directories will not be generated when the directories are generated

**Comment by mingyuecall** [comment]
Hi Travis,

Thank you very much.

Please feel free to check my homepage at mchen.fun

As well as my notion page
https://decorous-resistance-af5.notion.site/Ming-Chen-88d0aba742d8436cab57db97e5f94fd9

Best regards
Eric
2022年3月29日 +0800 22:54 Travis Fischer ***@***.***>，写道：
